jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/diagram/SubProcessInstanceItemView.html

```diff
@@ -0,0 +1,13 @@
+<li data-field="list-item">
+        <tr>
+            <div>
+                <a class="btn-link form-control-static" data-field="sub-process-name-text" id="subProcessNameText"></a>
```

Comment:
To keep consistency with the process breadcrumb, I suggest to instead of breaking line to show the ID, please use the format of Process Instance Id - Process Name. Ex: 2 - place-order